clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestAdminTool.java

```diff
@@ -74,13 +74,15 @@ public void testAdminToolRequiresSSLConfigFile() {
   public void testAdminUpdateStoreArg() throws ParseException, IOException {
```

Comment:
Seems this is a unit test?

Reply:
I was told that it was an integration test

Reply:
Yeah I did not checked how it was implemented before I told you to add/modify an integration test, but the integration test usually spin up the whole venice cluster in the test suite and run end to end. This only creates admin tool object so it is not.

Reply:
oh ok